documentation/modules/ROOT/pages/payloads.adoc

```diff
@@ -174,8 +174,100 @@ A Property `errorMessages` with a Collection Characteristic and effective data t
 }
 ----
 
+[[characteristics-payload-mappings]]
+== Specific payload mappings for Characteristics
 
-CAUTION: Due to the https://www.ecma-international.org/ecma-262/5.1/#sec-8.5[limits in the
+This section describes how data payloads look like for specific xref:characteristics.adoc[Characteristics].
+
+|===
+| Characteristic Name | Corresponding JSON data type
+| `samm-c:Either` | A JSON object with exactly either a `left` or `right` key. If the `left` key is present, its data type is the effective data type of the Characteristic referred to by `samm-c:left`; if the `right` key is present, its data type is the effective data type of the Characteristic referred to by `samm-c:right`.
+|===
+
+
+The following example Aspect uses `samm-c:Either` to express that the Property `speedValue` can either return the integer value for speed,
+or an Error message (using an Entity), if the value could not be determined (cf. xref:modeling-guidelines.adoc#declaring-either[Declaring Either]).
+
+So, assuming an Aspect Model has a Property `result` with its `samm:characteristic` set to `:Result` according to the following definition:
+[source,turtle,subs="attributes+"]
+----
+include:ROOT:example$either-declaration.ttl[tags=either-declaration]
```

Comment:
Apparently two colons are needed to resolve the include correctly, my previous suggestion was wrong. This works:
```suggestion
include::ROOT:example$either-declaration.ttl[tags=either-declaration]
```